design/KafkaDesign.md

```diff
@@ -0,0 +1,420 @@
+# Kafka Documentation
+
+Kruize has a BulkService feature which designed to streamline the detection, management, and optimization of containerized environments. It relies on the REST APIs without a message broker, which introduces latency when handling multiple parallel requests.
+Kafka replaces this existing REST API service for faster, asynchronous communication and to ensure seamless handling of recommendations at scale.
```

Comment:
Kafka does not "replace" but instead augments it. The user is free to choose either Kafka or the REST API to handle the recommendations

Reply:
That's right. Modified the description now.